testing/tests/DevExpress.ui.widgets/filterBuilderParts/commonTests.js

```diff
@@ -0,0 +1,388 @@
+"use strict";
+
+/* global fields */
+
+var $ = require("jquery"),
+    fx = require("animation/fx");
+
+require("ui/filter_builder/filter_builder");
+
+var FILTER_BUILDER_CLASS = "dx-filterbuilder",
+    FILTER_BUILDER_ITEM_FIELD_CLASS = "dx-filterbuilder-item-field",
+    FILTER_BUILDER_ITEM_OPERATION_CLASS = "dx-filterbuilder-item-operation",
+    FILTER_BUILDER_ITEM_VALUE_CLASS = "dx-filterbuilder-item-value",
+    FILTER_BUILDER_ITEM_VALUE_TEXT_CLASS = "dx-filterbuilder-item-value-text",
+    FILTER_BUILDER_GROUP_CONTENT_CLASS = "dx-filterbuilder-group-content",
+    FILTER_BUILDER_IMAGE_ADD_CLASS = "dx-icon-plus",
+    FILTER_BUILDER_IMAGE_REMOVE_CLASS = "dx-icon-remove",
+    FILTER_BUILDER_GROUP_OPERATION_CLASS = "dx-filterbuilder-group-operation",
+    ACTIVE_CLASS = "dx-state-active";
+
+var clickByButtonAndSelectMenuItem = function($button, menuItemIndex) {
+    $button.click();
+    $(".dx-menu-item-text").eq(menuItemIndex).trigger("dxclick");
+};
+
+QUnit.module("Rendering");
```

Comment:
You can use tests inside a module, as in the following example:
`
Quint.module("Rendering", { 
    beforeEach: function(){},
    afterEach: function(){}
},
function() {
    Qunit.test("Test", function(){
     });
});
`